### PR TITLE
Improve GeoJSON functionality

### PR DESCRIPTION
**Pull request by mwayne**
These changes:

- Fix saving and plotting GeoJSON files
- Add support for adding, editing, or removing feature properties
- Includes a test to confirm basic feature manipulations are present in a saved geojson file

I've copied the json loader's `_rowdict` function, but I'm happy to refactor if you have a different way you'd like to handle that.

### CODE REVIEW

File: visidata/loaders/geojson.py [hunk 1]
```diff
@@ -36,7 +39,7 @@ def iterload(self):
                 for prop in feature.get('properties', {}).keys():
                     prop = self.maybeClean(prop)
                     if prop not in self.colnames:
-                        c = Column(name=prop, getter=getter_factory(prop))
+                        c = GeoJSONColumn(name=prop, prop=prop)
```
Comment (saulpw) [inline visidata/loaders/geojson.py:42]: VisiData typically uses `expr` for things like this, so that it's consistently editable on the Columns Sheet.

**Review by saulpw — CHANGES_REQUESTED** [review]
Hey @mwayne, this looks great, thanks for your diligence and effort!  If you can change `prop` to `expr` as suggested, then we can ask @anjakefala to give it a once-over and look through your test, and then I'm happy to merge it.

**Comment by anjakefala** [comment]
This PR has my seal of approval!

File: tests/golden/save-geojson.geojson [hunk 2]
```diff
@@ -0,0 +1 @@
....741169], [-117.151181, 32.741129], [-117.152596, 32.738939], [-117.156007, 32.736716], [-117.155981, 32.734443999999996], [-117.154678, 32.731418], [-117.164849, 32.731365], [-117.16486, 32.732423], [-117.168173, 32.732316], [-117.169871, 32.731808], [-117.169468, 32.73083], [-117.170765, 32.730423], [-117.171347, 32.731485], [-117.178514, 32.739661999999996], [-117.180216, 32.741413], [-117.182409, 32.742553], [-117.181565, 32.743007], [-117.179618, 32.745573], [-117.178259, 32.746624], [-117.179939, 32.748529999999995], [-117.179198, 32.749451], [-117.176816, 32.750374], [-117.179828, 32.750054999999996], [-117.183505, 32.746641], [-117.184116, 32.745587], [-117.185361, 32.746344], [-117.186408, 32.746998999999995], [-117.186767, 32.746691], [-117.187772, 32.746975], [-117.188188, 32.748247], [-117.189442, 32.749269999999996], [-117.19213500000001, 32.751090999999995], [-117.19229899999999, 32.752485], [-117.192065, 32.752747], [-117.193934, 32.753906], [-117.193809, 32.755016999999995], [-117.196635, 32.759118], [-117.19633, 32.759502999999995]]]}, "type": "Feature", "properties": {"A": 3, "city": "San Diego", "postal-code": "92104", "county_fips": "073", "state_fips": "06"}}]}
```
Comment (anjakefala) [inline tests/golden/save-geojson.geojson:1]: Thanks for paying careful attention to the name of your test! =)

**Pull request merged**